Big-Spender/readme.md

```diff
@@ -86,6 +97,13 @@ INSERT YOUR QUERY HERE
 ```sql
 CREATE YOUR QUERY HERE
 ```
+select sum (amount), date FROM spends GROUP BY date; 
+
+OR
+
+select sum (amount), date_trunc('month', date) FROM spends GROUP BY date;
+OR
+elect sum (amount), date_trunc('month', date) FROM spends GROUP BY date_trunc('month', date);
```

Comment:
So what would be the difference between these three? how do these work? 

Why does the first one work in this instance? IF we had 2021-04-20, would this still filter by month?